L 05/25/2021 - 16:25:57: SourceMod error session started
L 05/25/2021 - 16:25:57: Info (map "vsh_cairo_pyramid_v4") (file "/home/frenzoid/TF2_FF2/tf/addons/sourcemod/logs/errors_20210525.log")
L 05/25/2021 - 16:25:57: [SM] Exception reported: Invalid database Handle 0 (error: 4)
L 05/25/2021 - 16:25:57: [SM] Blaming: sbpp_sleuth.smx
L 05/25/2021 - 16:25:57: [SM] Call stack trace:
L 05/25/2021 - 16:25:57: [SM]   [0] SQL_TQuery
L 05/25/2021 - 16:25:57: [SM]   [1] Line 170, D:\MegaSync\Projects\Sourcemod\scripting\sbpp_sleuth.sp::OnClientPostAdminCheck
L 05/25/2021 - 16:25:57: [sbpp_comms.smx] Connecting to database failed: [2003]: Can't connect to MySQL server on 'oldbox.cloud' (111)
L 05/25/2021 - 16:27:57: Error log file session closed.
L 05/25/2021 - 16:29:58: SourceMod error session started
L 05/25/2021 - 16:29:58: Info (map "vsh_aquaticbase_b5") (file "/home/frenzoid/TF2_FF2/tf/addons/sourcemod/logs/errors_20210525.log")
L 05/25/2021 - 16:29:58: [SM] Exception reported: Invalid database Handle 0 (error: 4)
L 05/25/2021 - 16:29:58: [SM] Blaming: sbpp_sleuth.smx
L 05/25/2021 - 16:29:58: [SM] Call stack trace:
L 05/25/2021 - 16:29:58: [SM]   [0] SQL_TQuery
L 05/25/2021 - 16:29:58: [SM]   [1] Line 170, D:\MegaSync\Projects\Sourcemod\scripting\sbpp_sleuth.sp::OnClientPostAdminCheck
L 05/25/2021 - 16:29:58: [sbpp_comms.smx] Connecting to database failed: [2003]: Can't connect to MySQL server on 'oldbox.cloud' (111)
